library/ssl_tls.c

```diff
@@ -2500,6 +2535,9 @@ static void ssl_extract_add_data_from_record( unsigned char* add_data,
     }
 }
 
+#define ENCRYPTION_SUCCESS 0xCC
+#define ENCRYPTION_FAIL 0xAA
```

Comment:
@AndrzejKurek , does it add us some security to use predefined values instead of regular error codes in the function?
Why we don't use  
int encryption_status = BEDTLS_ERR_PLATFORM_FAULT_DETECTED;

Reply:
Added.